flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "NgiPkgs";
 
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
-  inputs.nix-php-composer-builder.url = "github:loophp/nix-php-composer-builder";
+  inputs.nix-php-composer-builder.url = "github:loophp/nix-php-composer-builder?ref=0caf5a60753397cfa959a74fc9ea0562556573c1";
```

Comment:
Do you have a particular reason to pin this input to specific rev without a ref? If you just require a newer version, use `nix flake lock --update-input nix-php-composer-builder`.

Reply:
Yes, from the PR description

```
We are pinning it to 0caf5a60753397cfa959a74fc9ea0562556573c1 because that's what we developed against. Currently the latest version puts build artifacts in directories that are not compatible with our module definition. We will look into pointing to the latest of that repo or use the upstreamed version introduced in https://github.com/NixOS/nixpkgs/pull/248184
```

Since we finally got to a state where things were working, we wanted to merge that as a baseline and then update it afterwards